configs/manager/sample.toml

```diff
@@ -243,3 +243,7 @@ enhanced-aiomonitor-task-info = false
 
 # Periodically sync container stat from Redis to the kernels.last_stat column.
 periodic-sync-stats = false
+
+
+[license]
```

Comment:
In the development environment, this setting will cause an error indicating that it cannot connect to the license server. 
Wouldn't it be better to either comment out this setting only in `install-dev.sh` or find an alternative approach?

Reply:
If the license-related setting exists, but the associated plugin doesn't exist, no error is thrown.
In practice, we've been using the license-related setting in agent for a while now.

Reply:
You're correct, please don't mind.